### ISSUE TITLE
Request for client_credentials token doesn't work

### ISSUE THREAD

**Issue by robgmills** [robgmills/spring-microservice-oauth#1]
Requesting the client_credentials token using the `app` client 

```
curl http://app:password@localhost:8080/oauth/token\?grant_type\=client_credentials
```

fails with the message:

> {"error":"server_error","error_description":"Request method 'GET' not supported"}

**Comment by soarpatriot** [comment]
same issue, help!!!

**Comment by robgmills** [comment]
@soarpatriot Wow.  I haven't looked at this in over a year now since I started a new job.  I've actually got some time this weekend so I'll look at freshening it up.  Stay tuned.

**Comment by robgmills** [comment]
So, still not sure why it's happening.  I didn't get the opportunity to look into this that I thought I'd get.  But this endpoint does work if you `POST` to it:

```
curl -X POST http://app:password@localhost:8080/oauth/token\?grant_type\=client_credentials
```

returns

```
{"access_token":"03a03e27-c873-4e45-a1c8-49873360183f","token_type":"bearer","expires_in":43199,"scope":"read write"}
```

**Comment by djmills** [comment]
Yeah, the error means it can't resolve the endpoint as a get, double check
that it has one implemented
